src/npg_porch_cli/api_cli_user.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2022, 2024 Genome Research Ltd.
+#
+# Author: Jennifer Liddle <js10@sanger.ac.uk>
+#
+# This file is part of npg_porch_cli project.
+#
+# npg_porch_cli is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+
+from npg_porch_cli.api import PorchRequest
+
+
+def run():
+    """
+    Parses command line arguments, send the request to porch server API,
+    prints out the server's response to STDOUT.
+
+    The text below assumes that this code is deployed a a script called
+    `npg_porch_client`.
+
+    One positional and one names argument to teh script are required.
```

Comment:
typo "teh", odd grammar.

Reply:
fixed

Reply:
There's another one in the following text.